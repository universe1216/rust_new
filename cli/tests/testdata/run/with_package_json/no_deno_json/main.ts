// TODO(bartlomieju): currently we don't support actual bare specifier
// imports; this will be done in a follow up PR.
// import express from "express";

// console.log(express);
console.log("ok");
