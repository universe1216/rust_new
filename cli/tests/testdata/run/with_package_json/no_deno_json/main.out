[WILDCARD]package.json file found at '[WILDCARD]with_package_json[WILDCARD]package.json'
[WILDCARD]
ok
[Chalk (anonymous)] {
  constructor: [Function (anonymous)],
  Instance: [Class: ChalkClass],
  supportsColor: false,
  stderr: [Chalk (anonymous)] {
    constructor: [Function (anonymous)],
    Instance: [Class: ChalkClass],
    supportsColor: false
  }
}
